engine/Dockerfile

```
@@ -0,0 +1,20 @@
+FROM python:3.12.12-slim
+
+COPY --from=ghcr.io/astral-sh/uv:0.10.7 /uv /usr/local/bin/uv
+
+RUN addgroup --system --gid 1001 appgroup \
+ && adduser --system --uid 1001 --gid 1001 appuser
+
+WORKDIR /app
+
+COPY pyproject.toml uv.lock .
+RUN uv sync --no-dev --frozen
+j
```

Hadolint (2.14.0)

[error] unexpected 'j'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)